src/app/test_executive/test_executive.ml

```diff
@@ -89,7 +90,8 @@ let () =
   let test_executive_term =
     Term.(
       const start $ inputs
-      $ const (module Block_production_test.Make : Test_functor_intf))
+      $ const (module Block_production_test.Make : Test_functor_intf)
+      $ const (module Send_payment_test.Make : Test_functor_intf))
```

Comment:
I think these are actually unused now that the `test` is selected via a position cli argument. You should just be able to do `Term.(const start $ inputs)` now.